prow/cmd/sinker/main.go

```diff
@@ -239,9 +265,20 @@ func (c *controller) clean() {
 			// Delete old finished or orphan pods. Don't quit if we fail to delete one.
 			if err := client.Delete(pod.ObjectMeta.Name, &metav1.DeleteOptions{}); err == nil {
 				c.logger.WithField("pod", pod.ObjectMeta.Name).Info("Deleted old completed pod.")
+				c.currentMetrics.podsTotalRemovedByProw++
+				// TODO(hongkliu) replace this with the real reason
+				c.currentMetrics.addPodByReason("unknown", c.logger)
 			} else {
 				c.logger.WithField("pod", pod.ObjectMeta.Name).WithError(err).Error("Error deleting pod.")
 			}
 		}
 	}
+
+	c.currentMetrics.finishedAt = time.Now()
+	c.logger.WithFields(logrus.Fields{
+		"podsCreatedByProw": c.currentMetrics.podsCreatedByProw,
+		"timeUsed": c.currentMetrics.finishedAt.Truncate(time.Second). Sub(c.currentMetrics.startAt.Truncate(time.Second)),
+		"podsTotalRemovedByProw": c.currentMetrics.podsTotalRemovedByProw,
+		"podsRemovedByProw":      c.currentMetrics.podsRemovedByProw,}).
+		Info("Show sinkerReconciliationMetrics.")
```

Comment:
Maybe it would be helpful to create the `Fields` in a method of `sinkerReconciliationMetrics`, like Tide does it: https://github.com/kubernetes/test-infra/blob/master/prow/tide/tide.go#L1314.

Reply:
Done